### ISSUE TITLE
[device action node] changeStateTo false not working 

### ISSUE THREAD

**Issue by thost96** [mwittig/node-red-contrib-pimatic#2]
Hi, i noticed the following problem:
I created the following test flow and wanted to change the state of an homeduino device using the changeStateTo function. TurnOn and TurnOff works fine.

![image](https://user-images.githubusercontent.com/5551251/33947368-7fab1e14-e024-11e7-801b-00a715a96eaa.png)

The "On" funtion works like a charm, but the "Off" action also returns an payload of true, which should be false. 

Here are both debug msg objects:
On:
```
{"payload":true,"success":true,"deviceId":"homeduino_sofalampe","action":"changeStateTo","time":1513179804893,"_msgid":"bdc9d67c.423628"}
```

Off:
```
{"payload":true,"success":true,"deviceId":"homeduino_sofalampe","action":"changeStateTo","time":1513179820124,"_msgid":"d27e9d9e.2d816"}
```

And here is the whole flow config:
```
[{"id":"2e72786a.43f558","type":"device action","z":"e9863530.4f9458","controller":"9ced8722.da2f88","deviceId":"homeduino_sofalampe","action":"changeStateTo","parameterName":"state","parameterValue":"true","messageOnError":true,"name":"","x":520,"y":120,"wires":[["5fb08c9d.ac2ac4"]]},{"id":"19c23dd9.ad5a42","type":"device action","z":"e9863530.4f9458","controller":"9ced8722.da2f88","deviceId":"homeduino_sofalampe","action":"changeStateTo","parameterName":"state","parameterValue":"false","messageOnError":true,"name":"","x":520,"y":180,"wires":[["a7843241.f5772"]]},{"id":"5fb08c9d.ac2ac4","type":"debug","z":"e9863530.4f9458","name":"","active":true,"console":"false","complete":"true","x":770,"y":120,"wires":[]},{"id":"df75bd84.e8af2","type":"inject","z":"e9863530.4f9458","name":"An","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"x":270,"y":120,"wires":[["2e72786a.43f558"]]},{"id":"a4008769.6c4a18","type":"inject","z":"e9863530.4f9458","name":"Aus","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"x":270,"y":180,"wires":[["19c23dd9.ad5a42"]]},{"id":"a7843241.f5772","type":"debug","z":"e9863530.4f9458","name":"","active":true,"console":"false","complete":"true","x":770,"y":180,"wires":[]},{"id":"9ced8722.da2f88","type":"controller","z":"","name":"PI3","protocol":"http","host":"127.0.0.1","port":"80","path":"","username":"xxx","password":"xxx"}]
```

**Comment by mwittig** [comment]
Thanks for reporting. This is a bug I am going to fix asap. You can work around the matter if you change the payload type of the "Aus" inject node to "boolean" and set the sent value to "false".

```
[{"id":"1a70639d.cf68bc","type":"device action","z":"a2fc604f.9bf18","controller":"106c2fd9.3e1af","deviceId":"homeduino_sofalampe","action":"changeStateTo","parameterName":"state","parameterValue":"true","messageOnError":true,"name":"","x":1397.01953125,"y":849.00390625,"wires":[["40c85849.cfd868"]]},{"id":"781c0ce5.fde3c4","type":"device action","z":"a2fc604f.9bf18","controller":"106c2fd9.3e1af","deviceId":"homeduino_sofalampe","action":"changeStateTo","parameterName":"state","parameterValue":"false","messageOnError":true,"name":"","x":1397.01953125,"y":909.00390625,"wires":[["9c9aa669.846ab8"]]},{"id":"40c85849.cfd868","type":"debug","z":"a2fc604f.9bf18","name":"","active":true,"console":"false","complete":"true","x":1647.01953125,"y":849.00390625,"wires":[]},{"id":"9211ef76.3246b","type":"inject","z":"a2fc604f.9bf18","name":"An","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"x":1147.01953125,"y":849.00390625,"wires":[["1a70639d.cf68bc"]]},{"id":"f9f59cb7.91194","type":"inject","z":"a2fc604f.9bf18","name":"Aus","topic":"","payload":"false","payloadType":"bool","repeat":"","crontab":"","once":false,"x":1147.01953125,"y":909.00390625,"wires":[["781c0ce5.fde3c4"]]},{"id":"9c9aa669.846ab8","type":"debug","z":"a2fc604f.9bf18","name":"","active":true,"console":"false","complete":"true","x":1647.01953125,"y":909.00390625,"wires":[]},{"id":"106c2fd9.3e1af","type":"controller","z":"","name":"PI3","protocol":"http","host":"127.0.0.1","port":"80","path":"","username":"xxx","password":"xxx"}]

**Comment by mwittig** [comment]
Released node-red-contrib-pimatic@1.2.0
